FreeRTOS/FreeRTOS-Cellular-Interface
# Update version number by release flow

Update version number by release flow

Description
-----------

- Update all *.[ch] files with version number `<DEVELOPMENT BRANCH>`.
- Update release flow to update version number accordingly.
- Update the manifest/doxygen config version.
- Disable cloning CMOCK by default, and added --checkout to clone CMOCK while doing unit test CI.
- Update change log.

Test Steps
-----------

Run the release script locally and version number in all files are updated.

Checklist:
----------



Related Issue
-----------

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
